docs/developer-docs/smart-contracts/maintain/import.mdx

```diff
@@ -39,39 +39,9 @@ If a service canister depends on other canisters, those dependencies should also
 
 If the canister is meant for personal use and not intended for others, the canister should not be `pullable`.
 
-If a canister Wasm is published for other developers to use the canister should not be `pullable` since the canister ID of the instance is not static. Users can test integrations locally and deploy them directly. An example of this canister type is the asset canister generated by dfx.
+If a canister's Wasm is published for other developers to use, then the canister should not be `pullable` since the canister ID of the instance is not static. Users can test integrations locally and deploy them directly using the Wasm file directly.
```

Comment:
```suggestion
If a canister's Wasm is published for other developers to use, then the canister should not be `pullable` since the canister ID of the instance is not static. Users can test integrations locally and deploy them using the Wasm file directly.
```

Reply:
"should not be pullable" might be a bit harsh. maybe the sentence should be restructured a little bit. or do we really NOT recommend to make a canister pullable if it is using a Wasm that is (re-)used by many other canisters? 🤔